dbus-python build missing python3 during build

### Describe the bug
A clear and concise description of what the bug is.

### Steps To Reproduce
Steps to reproduce the behavior:
1. nix build github:nixos/nixpkgs/nixos-unstable#legacyPackages.x86_64-linux.python27Packages.dbus-python 

### Expected behavior
Nix builds the derivation.

### Additional context
```
checking for python3... no
configure: error: Cannot find python3 in your system path
```

### Notify maintainers
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: python27Packages.dbus-python
# a list of nixos modules affected by the problem
module:
```

--- a/pkgs/development/python-modules/dbus/default.nix
+++ b/pkgs/development/python-modules/dbus/default.nix
@@ -23,6 +23,10 @@ buildPythonPackage rec {
     MACOSX_DEPLOYMENT_TARGET=10.16
   '' else null;
 
+  configureFlags = [
+    "PYTHON_VERSION=${lib.versions.major python.version}"
+  ];
+
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ dbus dbus-glib ]
     # My guess why it's sometimes trying to -lncurses.

